conferences/abstracts/LIBER-submission.md

```diff
@@ -0,0 +1,25 @@
+# Is this the way to reproducibility?
```

Comment:
Could you adjust this to the title of the event rather than the title of the submission? I've added a suggestion but maybe you could spell out what LIBER is for rather than just the acronym? Only if that makes sense 😸 

```suggestion
# LIBER conference 
```